binding.gyp

```diff
@@ -73,12 +73,15 @@
       'NAPI_VERSION=7'
     ],
     'dependencies': [
-      '<!(node -p "require(\'node-addon-api\').gyp")',
       'libvips-cpp'
     ],
     'variables': {
-      'runtime_link%': 'shared',
       'conditions': [
+        ['OS != "emscripten"', {
+          'runtime_link%': 'shared',
+        }, {
+          'runtime_link%': 'static',
+        }],
```

Comment:
nit: perhaps it would be neater to set `'runtime_link': 'static',` in the `variables` object within the `wasm-scripts/common.gypi` file?

Untested, but I suppose that would work according to:
https://gyp.gsrc.io/docs/InputFormatReference.md#providing-default-values-for-variables

Reply:
It would work, but I intentionally keep that file for built-in Node.js variables so that it can be upstreamed properly... is `runtime_link` built-in or custom sharp variable?

Reply:
Looking at https://github.com/nodejs/node/blob/main/common.gypi, it doesn't seem to be standard variable, so it should stay in `binding.gyp`.